src/swish/swish.js

```diff
@@ -0,0 +1,26 @@
+// 0. Get originKey
+getOriginKey().then(originKey => {
+    // 1. Create an instance of AdyenCheckout
+    const checkout = new AdyenCheckout({
+        originKey, // Mandatory. originKey from Costumer Area
+        environment: 'test',
+        amount: { currency: 'EUR', value: 1000 },
+        onAdditionalDetails: result => {
+            console.log(result);
+        },
+        onError: error => {
+            console.log(error);
+        },
+    });
+
+    const swishData = {
+        paymentMethod: {
+            type: 'swish'
+        },
+        returnUrl: '.'
```

Comment:
```suggestion
        countryCode: 'SE',
        amount: { currency: 'SEK', value: 1000 }
```

We are already adding a `returnUrl` in the `utils.js` file. For this sample code purpose we can override the `countryCode` and `amount` from the frontend (of course that should never be done in production).